docs/scos/dev/glue-api-guides/202204.0/managing-products/managing-products.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Managing products
+description: Describes the endpoints used to manage products
+template: concept-topic-template
+---
+
+This section contains the descriptions of the following endpoints used to manage products in Spryker Commerce OS:  
```

Comment:
Do the people reading our docs know they are reading about SCOS?

Reply:
I will leave this as is for now, as we discussed